starknet_py/net/client_test.py

```diff
@@ -24,3 +25,26 @@ def test_handle_rpc_error_server_error():
 
     with pytest.raises(ServerError, match="RPC request failed."):
         RpcHttpClient.handle_rpc_error(no_error_dict)
+
+
+@pytest.mark.parametrize(
+    "key, expected",
+    [
+        (0x0, "0x00"),
+        (0x12345, "0x012345"),
+        (0x10001, "0x010001"),
+        (0xFFAA, "0x00ffaa"),
+        (0xDE, "0x00de"),
+    ],
+)
+def test_get_rpc_storage_key(key, expected):
+    assert _to_storage_key(key) == expected
+
+
+@pytest.mark.parametrize(
+    "key",
+    [int(1e100), -1, 0x8FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF],
+)
+def test_get_rpc_storage_key_raises_on_non_representable_key(key):
+    with pytest.raises(ValueError, match="cannot be represented"):
+        _to_storage_key(key)
```

Comment:
Maybe one more test case for `ADDR_BOUND - 1` would be handy? It's used to calculate the storage_var address.